specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2016-05-01/storageaccounts.json

```diff
@@ -0,0 +1,441 @@
+            "get": {
+                "x-ms-examples": {
+                    "Returns the requested storage account.": {
+                        "$ref": "./Examples/StorageAccounts/Get.json"
+                    }
+                },
+                "tags": [
+                    "StorageAccounts"
+                ],
+                "operationId": "StorageAccounts_Get",
+                "description": "Returns the requested storage account.",
+                "parameters": [
+                    {
+                        "$ref": "storage.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/FarmIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AccountIdParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK -- The storage account has been returned.",
+                        "schema": {
+                            "$ref": "#/definitions/StorageAccount"
+                        }
+                    },
+                    "404": {
+                        "description": "NOT FOUND -- The storage account or farm was not found."
+                    }
+                }
+            },
+            "post": {
+                "x-ms-examples": {
+                    "Undelete a deleted storage account.": {
+                        "$ref": "./Examples/StorageAccounts/Undelete.json"
+                    }
+                },
+                "tags": [
+                    "StorageAccounts"
+                ],
+                "operationId": "StorageAccounts_Undelete",
+                "description": "Undelete a deleted storage account.",
+                "parameters": [
+                    {
+                        "$ref": "storage.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/FarmIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AccountIdParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK -- Undelete either has been performed or account was not deleted."
+                    }
+                }

```

Comment:
Is this action a long running operation? if yes please use `x-ms-long-running-operation` extension.

Reply:
I don't know.